<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paypanel</class>
 <widget class="QWidget" name="paypanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="imageView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>971</width>
     <height>741</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/background);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>301</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>فاکتور تسویه</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="productLabel_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>211</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>کل فروش</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="buyTotal">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>391</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>30000 تومان</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="productLabel_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>250</y>
     <width>211</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>کل پرداختی</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="payTotal">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>391</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:green;</string>
   </property>
   <property name="text">
    <string>10000 تومان</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="productLabel_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>400</y>
     <width>211</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>باقی مانده</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="remainTotal">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>391</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>20000 تومان</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="payBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>751</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:&quot;green&quot;;
color:&quot;white&quot;;</string>
   </property>
   <property name="text">
    <string>تسویه</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
